schemas/create-hook-request.yml

```diff
@@ -21,18 +21,23 @@ properties:
       title: "Cron Pattern"
       type: string
       description: |
-          Cron-like specification for when tasks should be created.  The pattern is
-          parsed in a UTC context.
-          See [cron-parser on npm](https://www.npmjs.com/package/cron-parser).
-          Note that tasks may not be created at exactly the time specified.
+        Cron-like specification for when tasks should be created.  The pattern is
+        parsed in a UTC context.
+        See [cron-parser on npm](https://www.npmjs.com/package/cron-parser).
+        Note that tasks may not be created at exactly the time specified.
     uniqueItems: true
     default: []
   expires:                {$const: expires}
   deadline:               {$const: deadline}
   task:                   {$ref: "http://schemas.taskcluster.net/hooks/v1/task-template.json"}
   triggerSchema:
+    description: |
+      The hook definition is a json-e template which is combined with a trigger
+      payload to form a task definition. The `triggerSchema` property defines a
+      json schema to validate the trigger payload against.
```

Comment:
Please add a link to the firing-hooks doc (see the API description at the top of this file) for the details of the format of this context, since the payload and the context aren't exactly the same, and the difference can be confusing :)

Reply:
@djmitche said the most important point here, payload and the context aren't exactly the same and is important avoid any confusion with this, I had a lot..